python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/experimental/conftest.py

```diff
@@ -27,7 +27,9 @@
 TEST_STREAM_NAME = "test_stream"
 TEST_SELECTED = True
 TEST_JSON_SCHEMA = {}
-TEST_JOB_ID = "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+TEST_JOB_ID = 12345
```

Comment:
Fixes an error in Airbyte's example in the API documentation [here](https://reference.airbyte.com/reference/getjob). The json example uses a UUID as the Job ID, but the documentation mentions that the ID is an int.